persist/fs/write.go

```diff
@@ -85,6 +86,7 @@ func (w *writer) Open(
 	blockStart time.Time,
 ) error {
 	shardDir := ShardDirPath(w.filePathPrefix, namespace, shard)
+	fmt.Println("shard dir path", shardDir)
```

Comment:
Can remove this yeah?

Reply:
yeah i caught a few others, this was actually causing the integration test to fail i think